ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -527,6 +527,13 @@ public void setAutoCapitalize(ReactEditText view, int autoCapitalize) {
 
   @ReactProp(name = "keyboardType")
   public void setKeyboardType(ReactEditText view, @Nullable String keyboardType) {
+    int passwordVisibilityFlag = InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD &
+        ~InputType.TYPE_TEXT_VARIATION_PASSWORD;
+
+    int keyboardTypeFlags = INPUT_TYPE_KEYBOARD_NUMBERED |
```

Comment:
Let's move this and passwordVisibilityFlag to private static final members in the class.

Reply:
Will do, commit incoming.